lib/crawly/pipelines/write_to_file.ex

```diff
@@ -57,22 +62,27 @@ defmodule Crawly.Pipelines.WriteToFile do
 
   # No active FD
   def run(item, state, opts) do
-    opts = Enum.into(opts, %{folder: nil, extension: nil})
+    opts =
+      Enum.into(opts, %{folder: nil, extension: nil, include_timestamp: true})
+
+    folder = Map.get(opts, :folder, "./")
 
-    folder =
-      Map.get(opts, :folder, "./")
+    :ok = maybe_create_folder(folder)
 
-    extension =
-      Map.get(opts, :extension, "jl")
+    extension = Map.get(opts, :extension, "jl")
 
-    fd = open_fd(state.spider_name, folder, extension)
+    filename = case  Map.get(opts, :include_timestamp, false) do
+      false -> "#{inspect(state.spider_name)}.#{extension}"
+      true ->
+        ts_string = NaiveDateTime.utc_now() |> NaiveDateTime.to_string()
+        "#{inspect(state.spider_name)}_#{ts_string}.#{extension}"
```

Comment:
i suggest using a `DateTime.to_unix/1` instead, as there are no special characters

Reply:
Are you talking about something like this:
```
 datetime = DateTime.now!("Etc/UTC") |> DateTime.to_unix |> to_string
"1589443961"
```
In general, we can do it, but it will be a bit less clear for the end-user?

Reply:
I agree. I think perhaps a nice solution would be to replace characters such that characters like #, space, :,. with underscores so that they don't make it difficult for doing scripting

Reply:
Sorry could you explain what should I do it terms of the code, as I am a bit confused.

Reply:
```
 ts_string = NaiveDateTime.utc_now() 
|> NaiveDateTime.to_string()
|> String.replace(" .,#:", "_")
```

Reply:
hm... I was not aware NaiveDateTime.utc_now()  could produce it..

Reply:
Oh sorry just checked the test assertion, saw wrongly.

```
# in the test
 ts = "2020-05-13 09:06:22.668828"
```
so it should replace the hyphens, colons, full stops, and spaces instead.
string replacement pattern should be " -.,:"

Reply:
Please check

Reply:
Looks good